torch/onnx/_internal/fx/export.py

```diff
@@ -15,11 +17,120 @@
 from torch.fx.experimental import proxy_tensor
 from torch.fx.passes import fake_tensor_prop
 from torch.nn.utils import stateless
+from torch.onnx import _type_utils
 from torch.onnx._globals import GLOBALS as ONNX_GLOBALS
-from torch.onnx._internal import jit_utils, registration
 from torch.utils import _pytree
 
 
+# A simple lookup table for atenlib functions
+_ATENLIB_FUNCTIONS = {
+    "aten::abs": ops.core.aten_abs,
```

Comment:
Consider rename `aten_abs` to `aten.abs` in another PR. @justinchuby, any comments?

Reply:
`aten.abs` will happen when we code gen the public interface for the atenlib. Since we don't have it now we are opting for a more explicit dictionary here.